features/polyExplorer/src/components/clusterStory/clusterStory.jsx

```diff
@@ -55,13 +57,17 @@ const ClusterStory = ({
                 className="content"
                 ref={scrollRef}
                 onScroll={() => handleProgress()}
-                style={
-                    fadingTopBackground
-                        ? {
-                              background: `linear-gradient( 180deg, ${fadingTopBackground.color} 0%, #FFFFFF ${fadingTopBackground.distance})`,
-                          }
-                        : null
-                }
+                style={{
+                    "--story-color-primary": primaryColor,
```

Comment:
Every story has a different value in this variable making it automatically adapting the color in eg. the title

Reply:
Great idea!